packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -246,20 +251,30 @@ export default {
 
     listItemClasses () {
       return [
-        'd-w100p',
-        'd-box-border',
-        'd-ps-relative',
-        'd-px16',
-        { 'd-bgc-secondary-opaque': this.isActive && this.state === DEFAULT_FEED_ROW_STATE },
-        FEED_ROW_STATE_BACKGROUND_COLOR[this.state],
         'dt-feed-item-row',
-        'd-t',
-        'd-tp-bgc',
+        { 'dt-feed-item-row--active': this.isActive && this.state === DEFAULT_FEED_ROW_STATE },
+        { 'dt-feed-item-row--state-transition': this.transitionActive },
+        FEED_ROW_STATE_BACKGROUND_COLOR[this.state],
+
       ];
     },
   },
 
+  watch: {
+    state (newState, oldState) {
+      if (newState !== DEFAULT_FEED_ROW_STATE) {
+        this.transitionActive = true;
+      }
+    },
+  },
+
   methods: {
+    transitionComplete () {
+      if (this.state === DEFAULT_FEED_ROW_STATE) {
+        this.transitionActive = false;
+      }
+    },
```

Comment:
Have to do this or it will cut out the transition instead of waiting for it to finish.